gadget-container/cleanup.sh

```diff
@@ -7,6 +7,10 @@ for i in ocihookgadget runc-hook-prestart.sh runc-hook-poststop.sh ; do
   /bin/rm -f /host/opt/bin/$i
 done
 
+# CRIO hooks
+/bin/rm -f /host/etc/containers/oci/hooks.d/gadget-prestart.json
+/bin/rm -f /host/etc/containers/oci/hooks.d/gadget-prestart.json
```

Comment:
prestart is repeated twice without poststop

Reply:
Solved.